src/io/flutter/server/vmService/frame/DartVmServiceEvaluator.java

```diff
@@ -56,17 +57,23 @@ public void evaluate(@NotNull final String expression,
       callback.errorOccurred("Device disconnected");
       return;
     }
+    final VmServiceWrapper vmService = myDebugProcess.getVmServiceWrapper();
+    if (vmService == null) {
+      // Not connected to the VM yet.
+      callback.errorOccurred("No connection to the Dart VM");
```

Comment:
This won't be visible to users, right?

Reply:
Not on hover. For hover if there is an error, no hover is displayed. This might be visible if you managed to hit this (race?) condition evaluating an expression in the watch window.
In any case, this is the same message the regular Dart IntelliJ plugin displays.